App.js

```diff
@@ -8,91 +8,58 @@ import {
 } from 'react-native'
 
 import PaddingTop from './App/Containers/PaddingTop'
+import WeatherBox from './App/Containers/WeatherBox'
 
 
 export default class App extends React.Component {
   constructor(props) {
     super(props)
   }
 
-  outputCubes = (obj) => {
-
-    test = (num) => {
-      switch(num){
-        case(1):
-          return 'Mon'
-          break;
-        case(2):
-          return 'Tue'
-          break;
-        case(3):
-          return 'Wed'
-          break;
-        case(4):
-          return 'Thu'
-          break;
-        case(5):
-          return 'Fri'
-          break;
-        case(6):
-          return 'Sat'
-          break;
-        case(7):
-          return 'Sun'
-          break;
-        default:
-          return '?'
-          break;
-      }
-    }
+  getWeatherBox = (obj) => {
+    return (
+      <WeatherBox
+        key={obj.key}
+        date={obj.date}
+        temp={obj.temp}
+        tempIcon={obj.tempIconPath}
+        weatherIcon={obj.weatherIconPath}
+      />
+    )
+  }
 
+  outputCubes = (obj) => {
+    const dayArr = [
+      [null],
+      [1, 'Mon'],
+      [2, 'Tue'],
+      [3, 'Wed'],
+      [4, 'Thu'],
+      [5, 'Fri'],
+      [6, 'Sat'],
+      [7, 'Sun']
+    ]
     return obj.map(function (d) {
-      const key  = d.dt
-      const temp = Math.floor(d.main.temp)
+      const weatherBoxData = {
+        key: d.dt,
+        temp: Math.floor(d.main.temp),
+        date: dayArr[(new Date(d.dt_txt).getDay())][0],
+        tempIconPath: './App/Assets/Icons/fahrenheit.png',
+        weatherIconPath: ''
+      }
```

Comment:
created this object instead of having individual variables to improve readability